wireapp/wire-android
# fix: rename group screen UI [AR-2788]

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-2788" title="AR-2788" target="_blank"><img alt="Bug" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10803?size=medium" />AR-2788</a>  [Edit conversation name] Triggering a system message only upon change, wrong label & title
  </td></table>
  <br />
 








----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

- system notification is triggered about a change in the group name even if the name is the same
- the button on the bottom has the wrong label (“Continue”)
- the button on the bottom is enabled even if there is no change
- the “group name” page has the wrong title (should be “Group Name”, not the actual group name)
- the input is not centered on the screen
- the screen is not scrollable on smaller screens

### Solutions

Fix all of the issues listed above.

### Testing

#### How to Test

Open "edit name" screen to change the name or "create new group" screen to see how it behaves with the additional dropdown.

### Attachments (Optional)

#### Before:

https://user-images.githubusercontent.com/30429749/209832098-d9706587-5de8-4009-afaf-d0f8d29e3537.mp4

#### After:

https://user-images.githubusercontent.com/30429749/209832152-092cfd3f-9371-4040-becc-31e1cdd04d9c.mp4

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
